src/System.Net.WebSockets.Client/tests/ClientWebSocketTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Tests;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.WebSockets.Client.Tests
+{
+    /// <summary>
+    /// ClientWebSocket tests that do require a remote server.
+    /// </summary>
+    public class ClientWebSocketTest
+    {
+        public readonly static object[][] EchoServers = WebSocketTestServers.EchoServers;
+
+        private const int s_TimeOutMilliseconds = 2000;
+
+        private readonly ITestOutputHelper _output;
+
+        public ClientWebSocketTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Theory, MemberData("EchoServers")]
+        public async Task EchoBinaryMessage_Success(Uri server)
+        {
+            if (WebSocketHelper.ShouldSkipTestOSNotSupported(_output)) return;
```

Comment:
To be honest, I usually would have formatted like that (since I tell others to do so).

But it just seemed so wasteful in white space.  What I really want is an Xunit attribute that can be added via some [Skip] attribute with PlatformID.Windows7.  Or even better something more dynamic that does the same check for 'PlatformNotSupportedException'.

So, I can change the formatting to conform to our own rules, but I view it as temporary until we getting "skipping" capability in our tests.